css/css-contain/content-visibility/content-visibility-050.html

```diff
@@ -28,10 +28,11 @@
     focusable.tabIndex = 0;
     container.appendChild(focusable);
     focusable.focus();
-    requestAnimationFrame(() => {
-      t.step(() => assert_greater_than(document.scrollingElement.scrollTop, 500));
-      t.done();
-    });
+    step_timeout(finish, 0);
+  }
+  function finish() {
+    t.step(() => assert_greater_than(document.scrollingElement.scrollTop, 500));
+    t.done();
```

Comment:
I think we should try following this pattern if possible: https://github.com/web-platform-tests/wpt/blob/611e18d18cc233b5e033a2034cb8b117b02dd685/css/css-scroll-snap/scroll-target-margin-005.html#L27-L30